src/UseCases/ApplyForMembership/ApplyForMembershipUseCase.php

```diff
@@ -70,7 +65,8 @@ public function applyForMembership( ApplyForMembershipRequest $request ): ApplyF
 			$application->cancel();
 		}
 
-		$application->notifyOfFirstPaymentDate( $this->paymentDelayCalculator->calculateFirstPaymentDate()->format( 'Y-m-d' ) );
+		// assuming this is a leftover and can go away
+		//$application->notifyOfFirstPaymentDate( $this->paymentDelayCalculator->calculateFirstPaymentDate()->format( 'Y-m-d' ) );
```

Comment:
This is a new concept that we'll have to move as an optional parameter into the payment creation request and handle it in the payment domain.